opr/grader.go

```diff
@@ -304,6 +303,11 @@ func (g *QuickGrader) Sync() error {
 }
 
 func (g *QuickGrader) FetchOPRBlock(block *EntryBlockMarker) (*OprBlock, error) {
+	// We want to set the OPR version for the network. This is kinda hacky, but the marshal json
```

Comment:
Agree with the hacky command 😄 It probably means there is a design problem somewhere. But not something to fix in this PR anyway

Reply:
Yea, I ended up passing context to the marshal json function via another route. I will adjust this comment, as it's still used in the polling library. We can remove this once v2 is activated

Reply:
Oh I lied. I actually passed the OPRVersion to the polling by adjusting the method definition. I can remove this and that global. Forgot to clean that up :+1: